src/components/SButton.vue

```diff
@@ -1,15 +1,28 @@
 <script>
-export default {
-  name: 'SButton'
-}
+    export default {
+      name: 'SButton'
+    }
+
 </script>
 
 <template>
-    <button>
-        <slot name="value"/>
+    <button class="btn">
+        <slot/>
     </button>
 </template>
 
 <!-- TODO: Style button -->
 <style lang="scss" scoped>
+
+$btn-bg: #ffff;
+$btn-border: #081f36;;
+.btn {
+    padding: 12px 12px;
+    background-color: $btn-bg;
+    border: 2px solid $btn-bg;
+}
+
+.btn:hover {
+    border: 2px solid $btn-border;
+}
 </style>
```

Comment:
Extra space on line 16.

Also, I like that you're reaching out for scss variables, but they're best used if in a central `.scss` file and then used across different files.

Probably a little bit out of scope for this project, but I'm glad you came across them.

Also, there is an important difference between `sass` and `css` variables, and probably in cases like this a `css` variable would be better used. `sass` are pre-compile, so you can't reassign them while the page is active, while `css` vars you can reassign, so are more flexible.

Even though sometimes you might want something inflexible. Almost like `let` vs `const`.

Reply:
Got it, just wanted to try them out! Removed SCSS variables for this project.